src/modules/submissions.js

```diff
@@ -2604,44 +2606,64 @@
 
 						shouldTeahouse.resolve( !hasTeahouseCat );
 					} );
-				}
+					// Check if the user has already been welcomed.
+					if ( data.sendWelcomeDraft ) {
+						userTalk.getCategories( /* useApi */ true ).done( ( categories ) => {
+							let hasWelcomeDraftCat = false,
+								welcomeDraftCategories = [
+									'Category:Wikipedians who have received a AfC welcome message'
```

Comment:
Consider changing to `Category:Wikipedians who have received an AfC welcome message`. I've started https://en.wikipedia.org/wiki/Wikipedia:Categories_for_discussion/Log/2024_November_7#Category:Wikipedians_who_have_received_a_AfC_welcome_message onwiki to get the category moved.